### ISSUE TITLE
research ticket: editor in immersive mode

### ISSUE THREAD

**Issue by kfarr** [3DStreet/3dstreet-editor#397]
https://twitter.com/mrdoob/status/1758531748483485842

appears to use
https://github.com/mrdoob/three.js/blob/dev/examples/jsm/interactive/HTMLMesh.js

**Comment by vincentfretin** [comment]
Yes indeed, looks like HTMLMesh, for aframe see https://github.com/AdaRoseCannon/aframe-htmlmesh
Also see https://github.com/c-frame/aframe-editor/issues/5

**Comment by Utopiah** [comment]
FWIW on Quest now the browser can be used while the immersive experience still runs and gets updated, cf e.g https://twitter.com/utopiah/status/1774435419125363119 so this could be an intermediary step.
